packages/fluentui/docs/src/examples/components/Card/Usage/CardExampleFocusableChildrenGrid.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  Flex,
+  Text,
+  Card,
+  Grid,
+  cardsContainerBehavior,
+  cardChildrenFocusableBehavior,
+  Button,
+  StarIcon,
+  DownloadIcon,
+  MoreIcon,
+} from '@fluentui/react-northstar';
+import { screenReaderContainerStyles } from '@fluentui/react-northstar';
+import * as React from 'react';
+
+type SampleCardProps = {
+  index?: number;
+};
+
+const SampleCard: React.FC<SampleCardProps> = ({ index, ...unhandledProps }) => {
+  return (
+    <Card accessibility={cardChildrenFocusableBehavior} aria-roledescription="user card" {...unhandledProps}>
+      <Card.Header>
+        <Text content={`User #${index}`} weight="bold" />
+      </Card.Header>
+      <Card.Footer>
+        <Flex space="between">
+          <Button content="Action" />
+          <Flex>
+            <Button icon={<StarIcon />} iconOnly text title="Favourite" />
+            <Button icon={<DownloadIcon />} iconOnly text title="Download" />
+            <Button icon={<MoreIcon />} iconOnly text title="More" />
+          </Flex>
+        </Flex>
+      </Card.Footer>
+    </Card>
+  );
+};
+
+const CardExampleFocusableGrid = () => {
+  const navigationMessageId = 'navigationMessage';
+  const defaultMessage = 'Press arrow keys to navigate between cards.';
+  const [navigationMessage, setNavigationMessage] = React.useState(defaultMessage);
+  const timeout = React.useRef<ReturnType<typeof setTimeout>>();
+  const handleGridFocus = () => {
+    console.log(navigationMessage);
+    if (navigationMessage !== null) {
+      timeout.current = setTimeout(() => {
+        setNavigationMessage(null);
+      }, 3000);
+    }
+  };
+
+  const handleGridBlur = e => {
+    if (!e.currentTarget.contains(e.relatedTarget)) {
+      setNavigationMessage(defaultMessage);
+    }
+  };
+
+  const cleanup = () => {
+    if (timeout.current) {
+      clearTimeout(timeout.current);
+    }
+  };
+
+  // make sure our timeout gets cleared if
+  // our consuming component gets unmounted
+  React.useEffect(() => cleanup, []);
+
+  return (
+    <>
+      <div id={navigationMessageId} style={screenReaderContainerStyles}>
+        {navigationMessage}
+      </div>
+      <Grid accessibility={cardsContainerBehavior} columns="3" onFocus={handleGridFocus} onBlur={handleGridBlur}>
+        <SampleCard index={1} aria-label="1 of 18" aria-describedby={navigationMessageId} />
```

Comment:
wouldn't it be simpler to use `_.times`?

Reply:
also in the case of the aria labels, does the user have context to know that he is on cards? so we can be sure that 1 of 18 means anything at this point.

Reply:
Yep, because cards have `aria-roledescription` attributes